enterprise/server/test/integration/build_logs/build_logs_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
We usually use `"github.com/golang/protobuf/ptypes"` (from `        "@com_github_golang_protobuf//ptypes:go_default_library_gen"`) and then `ptypes.TimestampProto` to get an instance of timestamp from a `time.Time`.

There's an open issue to migrate to `google.golang.org/protobuf`, I think, but for now we probably want to use the old package everywhere.

Reply:
I already introduced timestamppb and durationpb in earlier PRs. I think we can at least start to migrate to timestamppb and durationpb from ptypes.